packages/block-library/src/search/edit.js

```diff
@@ -123,10 +130,42 @@ export default function SearchEdit( {
 		};
 	};
 
+	const getInnerBorderRadiusStyle = () => {
+		if ( ! borderRadius ) {
+			return undefined;
+		}
+
+		if ( 'button-inside' === buttonPosition ) {
+			// Uses min value of 1 as having no radius when the outer does is jarring.
+			const radius = Math.max( borderRadius - DEFAULT_INPUT_PADDING, 1 );
+			return radius ? radius + 'px' : undefined;
```

Comment:
Template literals are the preferred approach to string concatenation in gutenberg codebase - https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#strings - ``${ radius }px``

Reply:
Thanks for pointing that out. I've fixed it now.